.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build install ISPC action
+
+
+permissions: read-all
+
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository to checkout (owner/repo)'
+        required: true
+        default: 'ispc/install-ispc-action'
+      branch:
+        description: 'Branch to checkout'
+        required: true
+        default: 'main'
+  pull_request:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Build action
+        run: |
+          npm install
+          npm run build
+          git diff --exit-code
```

Comment:
What does this do?

Is it to ensure that checked-in version is identical to the build? Is it sustainable without pinning `npm` version?

Reply:
Yes, exactly, e.g., it detects discrepancies like different licenses provided for the action in `package.json` and `package-lock.json` of #1 .

I am not sure whether it doesn't depends from `npm`/`nodejs` version completely but I see no references to their versions in `package.json`/`package-lock.json`. The generated result depends on `esbuild` and `actions` toolkit. Although they can depend on something provided by `nodejs`. So, I don't know that.

Reply:
Let's keep it this way for now and change in future if needed.